### DISCUSSION TITLE
Mac deployment requires "interface" section.

### DISCUSSION THREAD

**Discussion by DavidBC71** [chadgeary/cloudblock#44] [Q&A] [answered]
All works great on Windows and iPhone (thanks) but when trying to import conf file into client on Mac receive an error saying 

Configuration must have an "interface" section 

Any ideas?

**Comment by chadgeary** [comment, marked as answer]
Hey - does your .conf file not include one? It should look something like this (an example peer1.conf):

```
[Interface]
Address = 172.19.0.2
PrivateKey = ThisWillBeALongString
ListenPort = 51820
DNS = 172.18.0.3

[Peer]
PublicKey = AnotherLongStringHere
Endpoint = 100.101.102.103:51820
AllowedIPs = 172.18.0.3/32,172.18.0.5/32
```

I don't have a mac with me right now to test with, but i will this week.

**Comment by DavidBC71** [comment]
The file must have been corrupted. It was blank when I checked. Have re-downloaded and working now. Thanks for your help and time. 

Cheers
